pkg/controller/infinispan/util/yaml.go

```diff
@@ -0,0 +1,15 @@
+package util
+
+import (
+	"gopkg.in/yaml.v2"
+)
+
+// ToYaml converts a type instance into serialized yaml
+func ToYaml(in interface{}) ([]byte, error) {
```

Comment:
Can't we just call yaml funcs directly?

Reply:
Hmm, maybe... 

I was trying to isolate the use of them in case we end up using a different third party library at some point (e.g. go-yaml 3). I think it provides some good encapsulation, but I can remove it if it's unclear.